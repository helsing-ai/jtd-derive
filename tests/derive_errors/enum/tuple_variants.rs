#[derive(jtd_derive::JsonTypedef)]
enum Tuple {
    Foo(u32),
    Bar(),
}

#[derive(jtd_derive::JsonTypedef)]
enum MixedWithTuple {
    Foo { x: String },
    Bar(u32),
}

fn main() {}
