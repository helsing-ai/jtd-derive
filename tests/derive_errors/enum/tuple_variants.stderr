error: Typedef can't support tuple variants
 --> tests/derive_errors/enum/tuple_variants.rs:3:5
  |
3 |     Foo(u32),
  |     ^^^^^^^^

error: Typedef can't support tuple variants
  --> tests/derive_errors/enum/tuple_variants.rs:10:5
   |
10 |     Bar(u32),
   |     ^^^^^^^^
